mtp_bank_admin/settings/docker.py

```diff
@@ -32,6 +32,5 @@
     # strict-transport set at nginx level
     # SECURE_HSTS_SECONDS = 31536000
     # SECURE_HSTS_INCLUDE_SUBDOMAINS = True
-    SECURE_BROWSER_XSS_FILTER = True
```

Comment:
i suppose we could have moved this into the comment block above given that it still *is* enforced at the ingress level but also this is fine because the option no longer exists at all in django

Reply:
> but also this is fine because the option no longer exists at all in django

yeah that was the rational for removing it: This no longer a thing in Django (even tho a similar thing happens at Ingress level)